pretyping/recordops.ml

```diff
@@ -107,6 +109,34 @@ let find_projection = function
   | ConstRef cst -> Cmap.find cst !projection_table
   | _ -> raise Not_found
 
+let prim_table =
+  Summary.ref (Cmap_env.empty : Projection.Repr.t Cmap_env.t) ~name:"record-prim-projs"
```

Comment:
Not sure if this should be `Cmap`  or `Cmap_env`.
It's replacing the Environ so I'm going to use `Cmap`.
It probably doesn't matter anyways.

Reply:
As a rule of thumb, I think we should **always** use the `_env` version by default. The other one relies on crazy quotients that most of the time do not really make sense. Unluckily most of the codebase does the exact converse because the name is shorter.